app/src/main/res/values/strings.xml

```diff
@@ -480,4 +480,14 @@
   <string name="developer_options_force_crash">Force crash</string>
   <string name="all">All</string>
   <string name="mark_completed_uppercase">MARK COMPLETED</string>
+  <!-- ThirdPartyDependencyListActivity -->
+  <string name="third_party_dependency_list_activity_title">Third-party Dependencies</string>
+  <string name="third_party_dependencies">Third Party Dependencies</string>
```

Comment:
This name is really non-specific. Where is it used? That should be captured in the name. Ditto for the next string.

Reply:
The right string is `Third-party Dependencies` only, therefore removed this string.